src/locales/en.json

```
@@ -74,13 +74,13 @@
   "about_the_team_top_label": "About",
   "about_the_team_title": "Liker Land Team",
   "about_the_team_description": "Liker Land is an award-winning blockchain company that builds user-centric protocols and applications in Web3. The awardee of the Google News Innovation Challenge, AppWorks incubatee, and several hackathons winners. Liker Land is currently the maintainer of LikeCoin to provide infrastructure support. Validators of Cosmos, Osmosis, and Desmos.",
-  "tooltip_coming_soon":"Coming Soon",
-  "tooltip_signin":"Please Sign in First",
-  "tooltip_no_nft":"No owned NFTs to transfer",
-  "snackbar_error_insufficient":"Oops, Insufficient Balance.",
-  "snackbar_success_transfer":"Complete the transfer of NFT.",
-  "snackbar_success_collect":"Complete the collect of {NFT}.",
-  "snackbar_error_buyLIKE":"buy LIKE",
+  "tooltip_coming_soon": "Coming Soon",
+  "tooltip_signin": "Please Sign in First",
+  "tooltip_no_nft": "No owned NFTs to transfer",
+  "snackbar_error_insufficient": "Oops, Insufficient Balance.",
+  "snackbar_success_transfer": "Complete the transfer of NFT.",
+  "snackbar_success_collect": "Complete the collect of 《{NFT}》.",
```

Prefer normal double quote over Chinese double quote
```suggestion
  "snackbar_success_collect": "Complete the collect of \"{NFT}\".",
```